src/components/shared/team/ProfileCard.jsx

```diff
@@ -4,13 +4,14 @@ import './/profileCard.css';
 function ProfileCard({ img, name, title }) {
   return (
     <>
-      <div className='card_container'>
-        <div className='card_information_container'>
-          <div>
-            <img className='profile_img' src={img} alt='name' />
-          </div>
-          <div className='name_title'>{name}</div>
-          <div className='exec_title'>{title}</div>
+      <div className= 'flex w-[23.8rem] h-[13.0rem] bg-[rgb(43,57,80)] rounded-[1.25rem] px-[0.8rem] py-[1.5rem] gap-[1.25rem]'>
+        <div className='w-[9.8rem] h-[10.1rem] rounded-[5.7rem] overflow-hidden'>
```

Comment:
hmm is there anything else we could try to make the image fit?
I just generated this off GPT
<div class="aspect-w-1 aspect-h-1 overflow-hidden">
  <img src="path/to/your/image.jpg" alt="Face" class="object-cover object-center w-full h-full" />
</div>
if not then we can make sure that the images going in have an aspect ratio of 1:1